docs/administrator/security/verify-artifacts.md

```diff
@@ -153,3 +153,80 @@ If the echo is as above, it shows that software components and dependencies in t
 ```shell
 $ trivy sbom sbom-karmada.spdx --ignore-unfixed
 ```
+
+
+## Verification of artifacts with SLSA attestations
+
+### Prerequisites
+
+You need to install the following tools:
+
+- `slsa-verifier` ([Installation Guide](https://github.com/slsa-framework/slsa-verifier?tab=readme-ov-file#installation))
+
+### cli
+
+A single attestation (`karmada-cli.intoto.jsonl`) from each release is provided since release 1.10.3. This can be used with [slsa-verifier](https://github.com/slsa-framework/slsa-verifier) to verify that a CLI binary was generated using Karmada workflows on GitHub and ensures it was cryptographically signed.
+
+```shell
+slsa-verifier verify-artifact karmadactl-darwin-arm64.tgz \
+--provenance-path karmada-cli.intoto.jsonl \
+--source-uri github.com/karmada-io/karmada \
+--source-tag v1.10.3
+```
+
+If you only want to verify up to the major or minor version of the source repository tag (instead of the full tag), use the --source-versioned-tag which performs semantic versioning verification:
```

Comment:
```suggestion
If you only want to verify up to the major or minor version of the source repository tag (instead of the full tag), use the --source-versioned-tag, with which you can verify the semantic versions:
```